pkg/util/feature/OneAgnetConfigMapSample.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: operator-oneagent-features
+data:
+
+#EvaluationContext: per OneAgent version a file mount?
```

Comment:
The original requirement is to have one configMap for everything. Also we don't want to update configMap each time when oneAgent team will release. So better to have something similar we have inside node_controller and `CacheEntry`. To review it you can describe `dynatrace-node-cache`.

Reply:
- Also, it would be nice to have an example of integrating it with the current code base.?
- It would be nice to understand how we want to reconcile changes. Mount file will work, and k8s will update automatically. But how will the operator react to changes? It's unclear.

Reply:
to the first requirement: it might be simpler to separate different things and have an own entity for OneAgent configurations. So changes from one do not interfere with the other.

Will look into node_controller `CacheEntry`

currently I would try to create an object upon reconciliation with a lazy reading of available versions (via file listing) and reading a config if accessed.

I am uncertain what the downsides of this approach are (performance, some K8s behaviour I am ignorant of) but it is the simplest approach that came to my mind...